### ISSUE TITLE
Option to download the arXiv pdf when using `pubs add -X`

### ISSUE THREAD

**Issue by benureau** [pubs/pubs#158]
To be work on after #155 has been merged.

Ok, this one should be straightforward, but care needs to be taken with it. For instance, if we end up switching to the DOI, we should not download the arXiv pdf, or at least ask the user what to do. Also, it needs to optional, since the pdf might be to heavy for the connection, or the user might wish to only have lightweight references.

**Comment by boutil** [comment]
Something like this would be great. Being able to fetch metadata and the file in one operation (with an additional switch on the command line) would be a nice improvement for my workflow.

**Comment by StanczakDominik** [comment]
#184 started some work on this. However, I'm thinking about submitting a PR for this using https://pypi.org/project/arxiv/. Would that be fine to add as a lightweight dependency?

**Comment by benureau** [comment]
@StanczakDominik: Yes. Dependencies decrease the code we have to maintain for the project. I think that while #184 started to work, most of the work remained to make it robust and there was little point to rewrite it from scratch.

**Comment by benureau** [comment]
> Something like this would be great. Being able to fetch metadata and the file in one operation (with an additional switch on the command line) would be a nice improvement for my workflow.

I agree.
